Quaver.Shared/Screens/Settings/SettingsDialog.cs

```diff
@@ -309,7 +309,26 @@ private void CreateSections()
                     new SettingsSlider(this, "Effect Volume", ConfigManager.VolumeEffect, x => $"{x}%"),
                     new SettingsBool(this, "Pitch Audio With Rate", ConfigManager.Pitched),
                     new SettingsSlider(this, "Global Audio Offset", ConfigManager.GlobalAudioOffset, x => $"{x} ms"),
-                    new SettingsCalibrateOffset(this, "Calibrate Offset")
+                    new SettingsCalibrateOffset(this, "Calibrate Offset"),
+                    new SettingsSlider(this, "Audio Device Period", ConfigManager.DevicePeriod, x => $"{x} ms"),
+                    ((Func<Drawable>)(() =>
```

Comment:
Instead of this, I would suggest creating a property for `Audio Device Period` and overriding the `Audio Device Buffer Length` slider to take in the device period slider as an argument. Then you can hook onto its bindable like normal. If you do it this way, OnDestroyed won't be needed, and you can unsubscribe from the bindable directly in the slider's `Destroy()` method. 

If you have to expose a few properties in the slider and make them public to make it happen, then I wouldn't mind it, but we should try to not do things inline like this for the sake of keeping the codebase clean